src/main/java/io/openshift/booster/service/FruitController.java

```diff
@@ -16,50 +16,37 @@
 
 package io.openshift.booster.service;
 
+import io.openshift.booster.exception.NotFoundException;
+import io.openshift.booster.exception.UnprocessableEntityException;
+import io.openshift.booster.exception.UnsupportedMediaTypeException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
```

Comment:
We shouldn't use this

Reply:
@gytis You mean the import that uses all annotations under `org.springframework.web.bind.annotation`?

Reply:
Yeah, * import is not the best practice normally. For example they're not allowed neither in Spring Boot nor WildFly repos and are blocked with style-check rules.

Reply:
Gotcha! Will fix ASAP

Reply:
Updated the PR with the specified fix